Content.Goobstation.Common/CCVar/CCVars.Goob.cs

```diff
@@ -238,6 +238,12 @@ public sealed partial class GoobCVars
     public static readonly CVarDef<bool> UseAdminOOCColorInBwoinks =
         CVarDef.Create("admin.bwoink_use_admin_ooc_color", true, CVar.SERVERONLY);
 
+    /// <summary>
+    ///     Discord Webhook for the station report
+    /// </summary>
+    public static readonly CVarDef<string> StationReportDiscordWebHook =
+        CVarDef.Create("StationReportDiscordWebHook", string.Empty, CVar.SERVERONLY);
```

Comment:
You need to follow the CVar structure of `[type].[cvar]` and this should be confidential so it is secret.
```suggestion
    public static readonly CVarDef<string> StationReportDiscordWebHook =
        CVarDef.Create("stationreport.discord_webhook", "", CVar.SERVERONLY | CVar.CONFIDENTIAL);
```